Remotion/ObjectBinding/Web/UI/Controls/IBocRowHeaderColumnDefinition.cs

```diff
@@ -0,0 +1,29 @@
+// // This file is part of the re-motion Core Framework (www.re-motion.org)
+// // Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// //
+// // The re-motion Core Framework is free software; you can redistribute it
+// // and/or modify it under the terms of the GNU Lesser General Public License
+// // as published by the Free Software Foundation; either version 2.1 of the
+// // License, or (at your option) any later version.
+// //
+// // re-motion is distributed in the hope that it will be useful,
+// // but WITHOUT ANY WARRANTY; without even the implied warranty of
+// // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// // GNU Lesser General Public License for more details.
+// //
+// // You should have received a copy of the GNU Lesser General Public License
+// // along with re-motion; if not, see http://www.gnu.org/licenses.
+// //
+using System;
+using Remotion.Web.UI.Controls;
+
+namespace Remotion.ObjectBinding.Web.UI.Controls
+{
+  /// <summary>
+  /// Defines the required API for allowing a <see cref="BocColumnDefinition"/> to act as a header in a <see cref="BocList"/> row.
+  /// </summary>
+  public interface IBocRowHeaderColumnDefinition : IControlItem
```

Comment:
@a-ctor I'm not completely happy with the name of this interface. It's meant to introduce the `IsRowHeader` flag and the only other name I can come up with is `IBocRowHeaderEnabledColumnDefinition`. A similar pattern exists with `IBocSortableColumnDefinition` and `IsSortable` but somehow, this feels more like it enables configuration while `IBocRowHeaderColumnDefinition` feels like it's always a row header, regardless of the flag. Thoughts?

Reply:
`Sortable` sounds better because it is an adjective maybe it works if we use an adjective for the `IsRowHeader` as well.

e.g. `IBocRowHeadedColumnDefinition`, `IBocHeadedColumnDefinition`

I am not really happy but these are the only names I could come up with

Reply:
Thanks for the ideas. To me, `rowheaded` doesn't work for me, maybe because it reads so much like 'bull-headed' :D How about `IBocRowHeaderSupportingColumnDefinition`? Or just admit defeat and make it `IBocColumnDefinitionWithRowHeaderSupport`?

Reply:
I prefer `IBocColumnDefinitionWithRowHeaderSupport` over `IBocRowHeaderSupportingColumnDefinition`. We could also use the "is" in the name `IBocIsRowHeaderColumnDefinition`. But the `WithRowHeaderSupport` name is probably best.